docs/app-dev-context/list-filter.md

```diff
@@ -0,0 +1,26 @@
+A List Filter is the SQL-like string that is provided as the parameter to an [Advanced Visibility](/concepts/what-is-advanced-visibility) List API.
```

Comment:
For the app dev guide, seems like this context page and the "advanced visibility" context page might be the same?